config/env.sample

```diff
@@ -0,0 +1,5 @@
+PORT=5005
+CERT_FILE="config/public-cert.pem"
+KEY_FILE="config/private-key.pem"
+KEY_PASS=KEY_FILE
+AUTH_TESSEL_OA2="https://client_key@auth.tessel.io"
```

Comment:
There are two levels of "keys" you need to consider for the OAuth server:

- the "client" — the proxy daemon in this case — needs to prove its identity to the OAuth server. @HarleyKwyn has set up the production config for this, the sample config should have dev environment defaults for easiest bootstrapping.
- each "user" has an API key that the client (i.e. proxy server) would like to validate. These API keys will come over the wire when a proxy tunnel is set up, and do not get configured here.

It seems like you are removing half of the "client" level authentication stuff, way beyond what #6 did, and I would be surprised [i.e. security issue with OAuth server!] if your dev setup still worked after ripping out the client secret handling as this PR does.